index.js

```diff
@@ -45,17 +46,20 @@ function createDirectoryContents (templatePath, newProjectPath, projectName) {
 
     if (stats.isFile()) {
       const contents = fs.readFileSync(origFilePath, 'utf8');
-      const result = contents.replace(/base-app/g, projectName)
-
-      if (file === '.npmignore') file = '.gitignore';
-
+      var result = "";
+      if (projectChoice === "rails") {
```

Comment:
This can be improve like this

```
if (xxxx === rails) {
 // custom code for rails
}

result = contents...
```
Also
/BaseApp/g could be a constant